episodes/what-is-an-image.md

```diff
@@ -160,12 +160,15 @@ detection
 - **Detector:** the device that detects the light to form the digital image 
 e.g. a CCD camera
 
-To briefly summarise, light is emitted from the sample (e.g. from a fluorescent 
-marker), then gathered and focused with the objective lens, before hitting the 
-detector. The detector is divided into many small, light sensitive areas - these 
-are physical pixels, that will correspond to the pixels in the final image. When 
-light hits one of these areas it is converted to one or more electrons, with 
-more light resulting in more electrons and a higher final value for that pixel.
+To briefly summarise for a fluorescence microscopy image:   
+an excitation light source (e.g. a laser) illuminates the sample, a fluorescent 
```

Comment:
Maybe a slight tweak of "an excitation light source (e.g. a laser illuminates the sample, and this light is absorbed by (or excites a?) fluorescent label, causing it to emit light..."

Reply:
I'll add this small change and then merge - thanks!